knowledge_repo/app/templates/index-feed.html

```diff
@@ -1,81 +1,72 @@
 {% extends "index-base.html" %}
 
-{% block inner_content %}
+{% block style_links %}
+{{ super() }}
+<link rel="stylesheet" href="{{ url_for('static', filename='css/pages/index.css')}}">
+{% endblock %}
+
+{% macro render_tag(tag, post_id) %}
```

Comment:
[question] what was the reasoning for this macro? we only use it in this place, and it's made this diff a little harder to read (since the functionality hasn't changed, you've just made this a function).